source/getstarted.html.erb

```diff
@@ -61,30 +61,47 @@ title: Get started
-            <p>Contact GOV.UK Pay on <a href="mailto:govuk-pay-support@digital.cabinet-office.gov.uk" data-click-events data-click-category="Content" data-click-action="Email link clicked">govuk-pay-support@digital.cabinet-office.gov.uk</a> to receive our agreements.</p>
+              <p>Your organisation needs to agree to GOV.UK Pay’s Memorandum of Understanding (for crown bodies) or Contract (for non-crown bodies). They include our Terms of Use.</p>
+              <p>When you sign in to your GOV.UK Pay account, you’ll find a link to the agreements in the page footer. You’ll need to accept the terms before you can take payments.</p>
+              <p>Contact GOV.UK Pay on <a href="mailto:govuk-pay-support@digital.cabinet-office.gov.uk" data-click-events data-click-category="Content" data-click-action="Email link clicked">govuk-pay-support@digital.cabinet-office.gov.uk</a> with any queries.</p>
           </div>
 
           <h2 class="task-list-section">
-              <span class="task-list-section-number">6.</span> Set up your Payment Service Provider
+              <span class="task-list-section-number">6.</span> Payment Service Providers (PSP)
           </h2>
           <div class="task-list-items">
-              <p>You will need to have a contract for Payment Gateway and Merchant Acquiring services with a Payment Service Provider, who will process the payments taken using GOV.UK Pay. We will support the suppliers listed on the relevant Crown Commercial Service framework: Barclays, Global Payments, Stripe, WorldPay.</p>
-              <div class="panel panel-border-wide">
-                  <p>If you don’t have one already, then avoid any procurement work by signing up with the Government Banking Service. They have an existing contract that’s open to central government and health sector organisations. Contact the Government Banking helpdesk at <a href="mailto:helpdesk.gbs@hmrc.gsi.gov.uk" data-click-events data-click-category="Content" data-click-action="Email link clicked">helpdesk.gbs@hmrc.gsi.gov.uk</a> if you’d like to learn more and sign up.</p>
-              </div>
-              <p>When you have a Payment Service Provider, you need to make sure it is set up to work with GOV.UK Pay. We have instructions for <a href="https://docs.payments.service.gov.uk/#switching-to-production" data-click-events data-click-category="Content" data-click-action="Internal link clicked">configuring your account</a> in our documentation. </p>
-              <p><a href="/support" data-click-events data-click-category="Content" data-click-action="Internal link clicked">Contact GOV.UK Pay</a> if you’d like more advice on contracting.</p>
+              <p>You’ll need to have a contract with a Payment Service Provider (PSP), who will process the payments taken using GOV.UK Pay. There are 3 ways to get a PSP.</p>
+              <h3 class="heading-small">1. GOV.UK Pay has procured, Stripe, as our PSP.</h3>
```

Comment:
Can’t say “GOV.UK Pay has procured, Stripe, as our PSP” — it doesn’t make sense

Reply:
Will ask Tracy

Reply:
It makes perfect sense if the commas are removed.

Reply:
changed it to 

> Use GOV.UK Pay's Payment Service Provider, Stripe.